content/de/contribute/_index.md

```diff
@@ -0,0 +1,110 @@
+
+### Neues GitHub Issue anlegen
+Gehe zu den [Github Issues](https://github.com/cncf/glossary/issues) des Glossar Repositories und klicke auf "New issue".
+
+![issues](/images/how-to/howto-01.png)
+
+Dort sind eine Vielzahl von Vorlagen zu sehen. Um einen neuen Begriff in deutscher Sprache vorzuschlagen, wähle "German Language Glossary Request".
+
+![templates](/images/how-to/howto-02.png)
+
+Füge den gewünschten Begriff ein, und klicke auf "Submit new issue". Falls du den Begriff lediglich vorschlagen wolltest, bist du jetzt bereits fertig! Falls du auch daran arbeiten möchtest, folge den nächsten Schritten. 
+
+![neues issue](/images/how-to/howto-03.png)
+
+Treten dem #glossary Kanal in dem CNCF Slack bei und teile  @Catherine Paganini, @jmo und @Seokho Son mit, dass du einen neuen Vorschlag eingereicht hast und daran arbeiten möchtest. Sie werden dir das GitHub-Issue zuweisen und allen anderen signalisieren, dass der Begriff bereits jemandem zugewiesen ist.
+
+Hier kannst du sehen, dass die ersten drei Begriffe verfügbar sind, während der letzte Begriff bereits jemandem zugewiesen wurde.
+
+![Begriff zuweisen](/images/how-to/howto-04.png)
+
+Beachte bitte, dass du jeweils nur an einen Begriff gleichzeitig arbeiten kannst. Falls du an mehreren Begriffen arbeiten möchtest, schließe bitte zuerst den einen ab, bevor du an dem anderen arbeitest. 
+
+### Begriff einreichen (Erstellen eines Pull Request)
+
+Vor dem Start bitte sicherstellen, dass der [Style Guide] (https://glossary.cncf.io/style-guide/) beachtet wird - das hilft dabei, die weiteren Interaktionen zu minimieren. Wie im Style Guide angegeben, empfehlen wir, mit einem Google- oder Word-Dokument zu beginnen. 
+
+Sobald der Begriff fertig ist, innerhalb der Registerkarte "Code" bitte auf "content" klicken ...
+
+![Inhalt](/images/how-to/howto-05.png)
+
+... danach auf "de" ... 
+
+![Sprachen-Ordner](/images/how-to/howto-06.png)
+
+... _TEMPLATE.md auswählen
+
+![Vorlage](/images/how-to/howto-07.png)
+
+den Inhalt kopieren ... 
+
+![Inhalt füllen](/images/how-to/howto-08.png)
+
+... und wieder eine Ebene höher in den "de" Ordner wechseln. Jetzt einfach auf "add file" und "create new file" auswählen.
+
+![Neue Datei anlegen](/images/how-to/howto-09.png)
+
+Füge den Namen des Begriffs in die URL ein (keine Großschreibung und kein Leerzeichen) mit dem Appendix ".md" (wenn die Vorschau nicht funktioniert, wurde das Appendix ".md" nicht korrekt hinzuzugefügt). Jetzt kann die ausgefüllte Vorlage eingefügt werden. Bitte beachten, dass GitHub Markdown verwendet wird um den Text zu formatieren (z. B. Hyperlink, fett, kursiv). Hier ein [Markdown Cheat Sheet](https://www.markdownguide.org/cheat-sheet/). Unter "Vorschau" kann man sehen wie der Text aussehen wird.
```

Comment:
Are you using the infinitive to avoid "du"? Don't think that's necessary, I'd say "Bitte beachte, dass GitHub..."